SPECS/php-pear/php-pear.spec

```diff
@@ -70,19 +60,21 @@ Requires:       php-posix
 Requires:       php-tokenizer
 Requires:       php-xml
 Requires:       php-zlib
-
 Provides:       php-pear(Console_Getopt) = %{getoptver}
 Provides:       php-pear(Archive_Tar) = %{arctarver}
 Provides:       php-pear(PEAR) = %{version}
 Provides:       php-pear(Structures_Graph) = %{structver}
 Provides:       php-pear(XML_Util) = %{xmlutil}
 Provides:       php-pear(PEAR_Manpages) = %{manpages}
-
 Provides:       php-composer(pear/console_getopt) = %{getoptver}
 Provides:       php-composer(pear/archive_tar) = %{arctarver}
 Provides:       php-composer(pear/pear-core-minimal) = %{version}
 Provides:       php-composer(pear/structures_graph) = %{structver}
 Provides:       php-composer(pear/xml_util) = %{xmlutil}
+BuildArch:      noarch
```

Comment:
Could you move this between the sources and build dependencies? Linter moving it here is one of the quirks we still have to deal with.